wrapanapi/systems/container/rhopenshift.py

```diff
@@ -1273,7 +1273,7 @@ def check_scale_value():
 
         else:
             raise ValueError(
-                "This name %s is not found among " "deployment configs or stateful sets", name
+                "This name %s is not found among deployment configs or stateful sets", name
```

Comment:
These are artifacts from previous pre-commit formatting runs, IIRC black used to do this. Nice that they're getting picked up and cleaned up automatically now.